StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -189,9 +197,20 @@ class EmbeddedPaymentMethodsView: UIView {
             delegate?.heightDidChange()
         }
     }
+    
+    // MARK: Internal functions
+    func resetSelectionToLastSelection() {
```

Comment:
1. `resetSelectionToLastSelection` looks super similar to `didTap` -- maybe we find a way to share some code?
2.  I am seeing a case where i instantiate embedded, saved card is selected, i click on "new card" then tap cancel, and my original selection is not preserved.

Reply:
Refactored #1 and fixed #2.